ros/src/tl_detector/tl_detector.py

```diff
@@ -113,27 +113,29 @@ def get_closest_waypoint(self, pose):
             int: index of the closest waypoint in self.waypoints
 
         """
-        
+
         # Very high value is set as as initial distance.
         closest_waypoint_dist = 100000
         closest_waypoint_ind = -1
 
         # Looping through base waypoints to find the one closest to the car.
         for i in range(0, len(self.waypoints.waypoints)):
```

Comment:
Another cool approach is:

for i, waypoint in enumerate(self.waypoints.waypoints):

Then you don't have to type self.waypoints.waypoints[i] later :)

Reply:
Yes, thanks for the tip, I'm switching too many programming languages :-)